tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -44,6 +44,40 @@ eslintTester.run('avoid-leaking-state-in-components', rule, {
       code: 'export default Component.extend({ classNames: [], classNameBindings: [], actions: {}, concatenatedProperties: [], mergedProperties: [], positionalParams: [] });',
       parserOptions: { ecmaVersion: 6, sourceType: 'module' },
     },
+
+    {
+      code: 'export default Foo.extend();',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    },
+    {
+      code: 'export default Foo.extend({ someProp: "example", init() { this.set("anotherProp", []) } });',
```

Comment:
Can you add a test for there being mixins as initial args before the object literal?